deploy-dev.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
wat is dis?
Mange referanser til syfomock

Reply:
ups, glemte å slette